src/content/docs/en/guides/integrations-guide/node.mdx

```diff
@@ -106,6 +106,32 @@ export default defineConfig({
 });
 ```
 
+
+### `experimentalStaticHeaders`
+
+<p>
+  **Type:** `boolean` <br />
+</p>
+
+If enabled, the adapter will serve the headers of the static pages using the `Response` object. 
+
+Here is the list of the headers that are added:
+- The CSP header of the static pages is added when CSP support is enabled.
```

Comment:
```suggestion
For example, when [experimental Content Security Policy](/en/reference/experimental-flags/csp/) is enabled, `experimentalStaticHeaders` can be used to add the CSP headers to the `Response` object instead of creating a `<meta>` element:
```